common/runtime/config.go

```diff
@@ -0,0 +1,36 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package runtime
+
+import (
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+// GetConfig fetches a value, first looking up $envKey. If that's not found, it tries
+// `gcloud config get-value $gcloudConfig`.
+func GetConfig(envKey, gcloudConfig string) string {
```

Comment:
It's not clear, without looking at the implementation, that this is fetching config using `gcloud config`. Docs and func name should be more descriptive.

Reply:
If you were using this and your IDE said `GetConfig(envKey, gcloudConfig string)`, and you wrote `GetConfig("ZONE", "compute/zone")`, where would you have assumed that "compute/zone" be used?

Reply:
I probably wouldn't know how to write `GetConfig("ZONE", "compute/zone")` in the first place :)